web/src/components/troubleshoot/PreflightsProgress.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import Loader from "../shared/Loader";
+import "@src/scss/components/AirgapUploadProgress.scss";
+import { getReadableCollectorName } from "../../utilities/utilities";
+
+export default class PreflightProgress extends React.Component {
```

Comment:
mind changing this to functional component because it's stateless?

Reply:
Yeah sure thing.